subscription-engine/kafka-ws/README.md

```diff
@@ -6,20 +6,20 @@ It was designed to be used in the context of dojot IoT Platform for allowing use
 
 ## **Overview**
 
-**Kafka WebSocket service** allows the users to retrieve conditional and/or partial data from a given dojot topic in a Kafka cluster. It works with pure websocket connections, so you can create websocket clients in any language you want as long as they support **RFC 6455**.
+**Kafka WebSocket service** allows the users to retrieve conditional and/or partial data from a given dojot topic in a Kafka cluster. It works with pure websocket connections, so you can create websocket clients in any language you want as long as they support [RFC 6455](https://tools.ietf.org/html/rfc6455).
 
 ### **Connecting to the service**
 
-The connection is done via pure websockets using the URI `/v1/websocket/:topic`. It is also possible to pass conditional and filter options as part of the URI. In the following sections, it is explained in details how to compose the URI to retrieve filtered and/or partial data from a given topic.
+The connection is done via pure websockets using the URI `/kafka-ws/v1/topics/:topic`. It is also possible to pass conditional and filter options as part of the URI. In the following sections, it is explained in details how to compose the URI to retrieve filtered and/or partial data from a given topic.
```

Comment:
I'm no sure if we should keep `kafka-ws` as part of the URI here because this is a documentation of the service and `kafka-ws` is not part of the service endpoint if I understood rightly.